playbooks/vars/defaults.yaml

```diff
@@ -56,8 +56,11 @@ rhcos_meta_url: https://raw.githubusercontent.com/openshift/installer/master/dat
 # Define rhcos_url to override use of rhcos_meta_url
 
 enabled_services:
-  - /usr/share/openstack-tripleo-heat-templates/environments/services/octavia.yaml
-standalone_role: /usr/share/openstack-tripleo-heat-templates/roles/Standalone.yaml
+  - /usr/share/openstack-tripleo-heat-templates/environments/services/neutron-ovn-sriov.yaml
+#  - /usr/share/openstack-tripleo-heat-templates/environments/services/octavia.yaml
```

Comment:
octavia probably does not need to be disabled. Just did for this run for speed. and clarity.

Reply:
yeah that is something we can do via the overrides, maybe in your PR you could add an example of overrides in https://github.com/shiftstack/dev-install/tree/master/example-overrides